card/mod/05_standard/set/right/password.rb

```diff
@@ -1,26 +1,34 @@
 
 include All::Permissions::Accounts
 
-view :editor do |args|
+view :editor do
   card.content = ''
-  autocomplete = (@parent && @parent.card.name=='*signin+*account') ? 'on' : 'off' #hack
+
+  # hack
+  autocomplete = if @parent && @parent.card.name == '*signin+*account'
+                   'on'
+                 else
+                   'off'
+                 end
   password_field :content, class: 'card-content', autocomplete: autocomplete
 end
 
-view :raw do |args|
+view :raw do
   '<em>encrypted</em>'
 end
 
-event :encrypt_password, on: :save, after: :process_subcards, changed: :content,
-    when: proc{ |c| !Card::Env[:no_password_encryptions] } do
-      # no_password_encryptions = hack for import - fix with api for ignoring events
-
-  salt = (left && left.salt)
-  salt = Card::Env[:salt] unless salt.present? # hack - fix with better ORM handling
+event :encrypt_password, on: :save, after: :process_subcards,
+                         changed: :content,
+                         when: proc { !Card::Env[:no_password_encryptions] } do
+  # no_password_encryptions = hack for import - fix with api for ignoring events
+  salt = left && left.salt
+  # hack - fix with better ORM handling
```

Comment:
Annotation keywords like `hack` should be all upper case, followed by a colon, and a space, then a note describing the problem.